augur/translate.py

```diff
@@ -421,7 +440,8 @@ def run(args):
 
     ## load features; only requested features if genes given
     features = load_features(args.reference_sequence, genes)
-    print("Read in {} features from reference sequence file".format(len(features)))
+    print_err("Read in {} features from reference sequence file".format(len(features)))
+    validate_features(features, args.reference_sequence)
```

Comment:
Should this validation be done automatically as part of `load_features()` (i.e. does it also apply to `augur ancestral`)?

Reply:
Yes, because `load_features` is only used in `augur ancestral` in the code-path which takes translated (AA) sequences, typically produced via `nextclade`. 

`ancestral` already checks the length is a multiple of 3, BTW:

https://github.com/nextstrain/augur/blob/4a31adac39077c02d727c4e09df540a3c840c81b/augur/ancestral.py#L455-L459